packages/devtools/frigg-cli/deploy-command/index.js

```diff
@@ -113,7 +113,7 @@ function validateAndBuildEnvironment(appDefinition, options) {
 
     // Try to use the env-validator if available
     try {
-        const { validateEnvironmentVariables } = require('@friggframework/devtools/infrastructure/env-validator');
+        const { validateEnvironmentVariables } = require('../../infrastructure/env-validator');
```

Comment:
Are you sure this is a good idea?

Reply:
Basically trying to release frigg-cli as its own package, while inside the devtools package, and while requiring the devtools package, was making the CD environment do a double deploy.

Ultimately because there were two installs of the devtools package (under root, and then under /frigg-cli/node_modules).

So just making the CLI part of the devtools package resolve that; which means everything is relative pathing 👍  the cli runs from here.

@d-klotz